### PR TITLE
Fix end portal, block entity, and end sky shaders on 1.17

### PR DESCRIPTION
**Pull request by IMS212**

### CODE REVIEW

File: src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java [hunk 1]
```diff
@@ -232,4 +232,24 @@
...
+
+	@Inject(method = RENDER, at = @At(value = "CONSTANT", args = "stringValue=destroyProgress"), locals = LocalCapture.CAPTURE_FAILHARD)
+	private void iris$EndBlockEntities(MatrixStack matrices, float tickDelta, long limitTime,
+									   boolean renderBlockOutline, Camera camera, GameRenderer gameRenderer,
+									   LightmapTextureManager lightmapTextureManager, Matrix4f matrix4f,
+									   CallbackInfo ci, Profiler profiler, Vec3d vec3d, double d, double e, double f,
+									   Matrix4f matrix4f2, boolean bl, Frustum frustum2, boolean bl3,
+									   VertexConsumerProvider.Immediate immediate) {
+		pipeline.setPhase(WorldRenderingPhase.OTHER);
+	}
```
Comment (coderbot16) [inline src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java:254]: I don't think that local capture is needed here. Also, the convention is `iris$startBlockEntities` afaik.

**Review by coderbot16 — CHANGES_REQUESTED** [review]

File: src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java (hunk 1 above)
Comment (IMS212) [reply src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java]: Fixed, would you mind reviewing again

**Review by coderbot16 — CHANGES_REQUESTED** [review]
It looks like you didn't remove the stuff after the callbackinfo, which would probably cause the mixin to fail and the game to crash?

**Comment by IMS212** [comment]
Fixed

**Comment by coderbot16** [comment]
Thanks!

**Pull request merged**